src/WebForms/Compilation/Polyfills.cs

```diff
@@ -27,6 +30,75 @@ internal static object ParseExpression(string fullResourceKey)
     {
         throw new NotImplementedException("ParseExpression");
     }
+
+    internal static object GetGlobalResourceObject(string classKey, string resourceKey)
+    {
+        return GetGlobalResourceObject(classKey, resourceKey, null /*objType*/, null /*propName*/, null /*culture*/);
+    }
+
+    internal static object GetGlobalResourceObject(string classKey,
+        string resourceKey, Type objType, string propName, CultureInfo culture)
+    {
+
+        IResourceProvider resourceProvider = GetGlobalResourceProvider(classKey);
+        return GetResourceObject(resourceProvider, resourceKey, culture,
+            objType, propName);
+    }
+
+    private static IResourceProvider GetGlobalResourceProvider(string classKey)
+    {
+        return null;
+
+#if PORT_GetGlobalResourceProvider
+        // throw new NotImplementedException();
```

Comment:
for times like this, use HttpRuntimeHelper.GetLogger<> to get a logger to message things out